### DISCUSSION TITLE
Problems with preconditioning

### DISCUSSION THREAD

**Discussion by mitasa** [flobz/psa_car_controller#1005] [General]
Hi,

Seems to me that the MyOpel App on Android is nowadays asking for PIN code confirmation every time I start preconditioning. This results in preconditioning command not working anymore via the psa car controller, as it does not support this behaviour. PSA car controller otherwise works fine, because getting the information does not require using PIN code.

**Comment by zutislo** [comment]
Same with myCitroen app. Though, it strangelly works there quite reliable. But still crappapp. Would much preffer it with ha.
